src/plugins/intel_gpu/tests/unit/fusions/fully_connected_fusion_test.cpp

```diff
@@ -1034,7 +1036,54 @@ TEST_P(fc_fp32_activation_prelu, basic) {
 }
 
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, fc_fp32_activation_prelu, ::testing::ValuesIn(std::vector<fully_connected_test_params>{
-    fully_connected_test_params{ CASE_FC_FP32_1, 3, 3 }
+    fully_connected_test_params{ CASE_FC_FP32_1, 2, 3 },
+    fully_connected_test_params{ CASE_FC_FP32_4, 2, 3 }
+}));
+
+class fc_fp32_activation_prelu_full_tensor : public FullyConnectedFusingTestOneDNN {};
+TEST_P(fc_fp32_activation_prelu_full_tensor, basic) {
+    auto p = GetParam();
+    create_topologies(
+        input_layout("input", get_input_layout(p)),
+        data("weights", get_mem(get_weights_layout(p))),
+        data("bias", get_mem(get_bias_layout(p))),
+        data("data", get_mem(get_output_layout(p))),
```

Comment:
the name `data` is a bit misleading. What about `slope` or `prelu_slope`? (for other case as well)
what does `prelu_full_tensor` mean here?